### DISCUSSION TITLE
Improve "starred" design

### DISCUSSION THREAD

**Discussion by rubenmoya** [outline/outline#1795] [Ideas]
The design for the starred and unstarred documents is pretty similar, maybe it would be nice to make the unstarred one just an outline, or add some color to the starred one.

![image](https://user-images.githubusercontent.com/905225/104127810-7635bb80-5364-11eb-8991-3eed1e5802e0.png)

**Comment by tommoor** [comment]
This has now been updated to a hollow design when unstarred, thanks for the nudge!

**Reply by rubenmoya** [reply]
Nice! Can't wait to see it!
